tests/integration/exported_procedure/resources/taste/work/controller/SDL/src/controller.pr

```diff
@@ -0,0 +1,53 @@
+/* CIF PROCESS (250, 150), (150, 75) */
+process Controller;
```

Comment:
We have two outputs, and we wait for both in any order. But, shouldn't the order be ensured? I assume that testing exported procedures with exported procedures could be a mess, BUT, if we are doing sporadic signals, then maybe the could carry a parameter that is increased in procedure and in transition, and very whether the correct value is received?

Reply:
changed, added parameter for procedure